components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -78,7 +78,7 @@ @interface BannerTypicalUseExampleViewController () <UITableViewDelegate, UITabl
 @property(nonatomic, strong) NSArray<BannerExampleUseInfo *> *exampleList;
```

Comment:
Can we annotate this example as available only on iOS 9 instead of the code itself? I realize this won't affect our catalog's display of the example, but neither does this current implementation. It will be more clear/simpler if we annotate the example itself rather than all of its code.

Reply:
Sure. Both implementations don't affect the display in this case. Switched to the simpler version in 660f6ecd6a832d90b62bc67e27331c7a524bbd00